.circleci/config.yml

```diff
@@ -27,6 +27,10 @@ jobs:
           paths:
             - ./node_modules
       
+      - run:
+          name: eslint syntax check
+          command: npm run pretest
+
```

Comment:
we need the manual step to run `pretest` / `lint` in circle, because circle runs `npm run test-report` instead of `npm test`, in order to save test reports inside the circle run.